src/transformers/modeling_tf_bart.py

```diff
@@ -433,10 +437,12 @@ def call(
 
             Tuple containing, encoded output of shape `(seq_len, batch, embed_dim)`, self_attn_weights, layer_state
         """
+        residual = x  # Make a copy of the input tensor to add later.
```

Comment:
don't like `x` => should be `hidden_states` IMO, but I guess we had this discussion often enough :D